openfisca_core/tools/test_runner.py

```diff
@@ -165,15 +166,15 @@ def runtest(self):
         finally:
             tracer = self.simulation.tracer
             if verbose:
-                self.print_computation_log(tracer)
+                self.print_computation_log(tracer, depth_max)
```

Comment:
This seems to break or interfere with the _verbose_ option, is it what we want?

Reply:
In that case, the whole feature could be just called _verbosity depth_.

Reply:
Yes the computation log is not produced if the verbose option is not triggered.
Ok for verbosity depth. Thanks !

Reply:
Ok let me change my suggestions.

Reply:
Please do !